product-cli/cmd/cluster/cluster.go

```diff
@@ -41,6 +41,12 @@ func NewCreateCommands() *cobra.Command {
 		SilenceUsage: true,
 	}
 
+	cmd.PersistentFlags().Int32Var(&opts.NodePoolReplicas, "node-pool-replicas", opts.NodePoolReplicas, "If 0 or greater, creates a nodepool with that many replicas; else if less than 0, does not create a nodepool.")
+	cmd.PersistentFlags().StringVar(&opts.PullSecretFile, "pull-secret", opts.PullSecretFile, "File path to a pull secret.")
+	cmd.PersistentFlags().StringVar(&opts.ReleaseImage, "release-image", opts.ReleaseImage, "The OCP release image for the cluster.")
+
+	_ = cmd.MarkPersistentFlagRequired("pull-secret")
```

Comment:
I have a few more args that we use that I've listed in this pr, https://github.com/openshift/hypershift/pull/2780/files 

The args I've added there are all the ones that we are either documenting for the kubevirt platform, or plan to document soon.